docs/zh_cn/model_zoo.md

```diff
@@ -9,9 +9,9 @@
 
 ## 数据统计
 
-- 论文数量: 18
+- 论文数量: 19
```

Comment:
A few more words, I think there is no need to set this line and this line below here.

Reply:
this will be updated in PR https://github.com/open-mmlab/mmselfsup/pull/599